packages/shared/lib/contexts/governance/actions/getAccountsParticipationEventStatusForEvent.ts

```diff
@@ -1,16 +1,13 @@
 import { get } from 'svelte/store'
+
 import type { ParticipationEventId, ParticipationEventStatus } from '@iota/wallet'
+
+import { IAccount } from '@core/account/interfaces'
 import { selectedAccount } from '@core/account/stores'
-import { IAccount } from '@core/account'
 
-export async function getAccountsParticipationEventStatusForEvent(
+export function getAccountsParticipationEventStatusForEvent(
     eventId: ParticipationEventId,
     account: IAccount = get(selectedAccount)
 ): Promise<ParticipationEventStatus> {
-    try {
-        const status = await account?.getParticipationEventStatus(eventId)
-        return status
-    } catch (err) {
-        return undefined
-    }
+    return account?.getParticipationEventStatus(eventId)
```

Comment:
because of this change, you need to add a try catch in `selected-participation-event-status.store`. 
If not, the questions load permanently on detail view